### ISSUE TITLE
Missing 'fiware/kurento-examples' docker image

### ISSUE THREAD

**Issue by smellman** [Fiware/tutorials.Media-Streams#10]
I just run `./services create`, I got an error.
```
WARNING: Some service image(s) must be built from source by running:
    docker compose build kurento-examples
Error response from daemon: pull access denied for fiware/kurento-examples, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
I was looking for this docker image in [Fiware's docker profile](https://hub.docker.com/u/fiware/?page=1) but I didn't find it.

**Comment by jason-fox** [comment]
This image is just a build of the official **Kurento** tutorials (included as submodules) which can be found here:

https://github.com/Kurento/kurento-tutorial-node.git
https://github.com/Kurento/kurento-tutorial-java.git
https://github.com/Kurento/kurento-tutorial-js.git

As suggested you can force the build directly if you are getting an error:

```console
docker compose build kurento-examples
```

This FIWARE Foundation tutorial just picks a selection of  context-data relevant selection from the official t **Kurento** tutorials

**Comment by smellman** [comment]
Thanks.
I create PR #11 to supports README document.

**Comment by fisuda** [comment]
@jason-fox,
This error has been also reproduced in my environment. This may be due to the missing of process to build the docker image `fiware/kurento-examples` in the `services` script or  the missing of the image in the FIWARE's docker repository.

Why have you deleted the [line #95](https://github.com/FIWARE/tutorials.Media-Streams/commit/673bcbedd7c2baf2ab295e75b0a436a4bb4e66e9#diff-ef1c4b45f2ed763283f8de890730defb2e7d1f8f0cc4214b740dd8cc5f10364dL95) for building the `fiware/kurento-examples` image in the `services` script at the commit 673bcbedd7c2baf2ab295e75b0a436a4bb4e66e9 for fixing the issue #7?

**Comment by hsakamoto123** [comment]
I got the same error in my environment.
I don't know how to avoid it.
Please tell me more about how to avoid it.

**Comment by hsakamoto123** [comment]
Any new information on how to avoid it?

**Comment by jason-fox** [comment]
The issue has been fixed by #11 - thanks to @smellman  😄 - I have belatedly merged the fix.

I have also enabled the **Run in GitPod** button to enable users to run this using a Linux-based cloud environment - that  should hopefully reduce incompatible O/S issues on Windows for example.
